lua/telescope/builtin/__internal.lua

```diff
@@ -712,7 +712,17 @@ internal.help_tags = function(opts)
   end
 
   local help_files = {}
-  local all_files = vim.api.nvim_get_runtime_file("doc/*", true)
+
+  -- get the current vim runtime path
+  local rtp = vim.o.runtimepath
+  -- extend the runtime path with all plugins not loaded by lazy.nvim
+  -- only if lazy.nvim is available
+  local lazy = package.loaded["lazy.core.util"]
+  if lazy and lazy.get_unloaded_rtp then
+    local paths = lazy.get_unloaded_rtp ""
+    rtp = rtp .. "," .. table.concat(paths, ",")
```

Comment:
if paths is empty we end up with trailing comma

Reply:
Revised commit to make sure `paths` is non-empty. Thanks!